wallet/wallet.go

```diff
@@ -5155,3 +5155,74 @@ func (w *Wallet) GetCoinjoinTxsSumbByAcct(ctx context.Context) (map[uint32]int,
 
 	return allTxsByAcct, nil
 }
+
+// GetVSPTicketsByFeeStatus returns the ticket hashes of tickets with the
+// informed fee status.
+func (w *Wallet) GetVSPTicketsByFeeStatus(ctx context.Context, feeStatus int) ([]chainhash.Hash, error) {
+	const op errors.Op = "wallet.GetVSPTicketsByFeeStatus"
+	tickets := make(map[chainhash.Hash]*udb.VSPTicket)
```

Comment:
don't `make` if you are going to overwrite this anyways. just declare the variable.